src/main/java/com/github/dockerjava/api/model/SELContext.java

```diff
@@ -0,0 +1,44 @@
+package com.github.dockerjava.api.model;
+
+/**
+ * The context mode of bound volume in SELinux.
+ * Host path <code>shared</code> - can be used by all containers, <code>private</code> - by only this container
+ *
+ * @see http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/
+ */
+public enum SELContext {
```

Comment:
Do you have some of such possible extensions in mind ? I doubt it is really possible with current format of SE context passing as a single flag to volume bind - they are modelled after AccessMode enum.